packages/twenty-front/src/modules/object-record/record-table/components/RecordTableEmptyState.tsx

```diff
@@ -0,0 +1,63 @@
+import { useNavigate } from 'react-router-dom';
+import { ThemeProvider, useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import isEmpty from 'lodash.isempty';
+import { IconSettings, THEME_LIGHT } from 'twenty-ui';
+
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+import { Button } from '@/ui/input/button/components/Button';
+import AnimatedPlaceholder from '@/ui/layout/animated-placeholder/components/AnimatedPlaceholder';
+import {
+  AnimatedPlaceholderEmptyContainer,
+  AnimatedPlaceholderEmptySubTitle,
+  AnimatedPlaceholderEmptyTextContainer,
+  AnimatedPlaceholderEmptyTitle,
+} from '@/ui/layout/animated-placeholder/components/EmptyPlaceholderStyled';
+
+const StyledEmptyStateContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  margin-top: ${({ theme }) => theme.spacing(8)};
+`;
+
+type RecordTableEmptyStateProps = {
+  objectMetadataItem: ObjectMetadataItem;
+};
+
+export const RecordTableEmptyState = ({
```

Comment:
This component looks like `GenericErrorFallback`. But not sure I should refactor to use the same base for both since this is not the same use case